src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
-// write code here
+const tBody = document.querySelector('tbody');
+const tr = tBody.querySelectorAll('tr');
+const thead = document.querySelector('thead');
+const th = thead.querySelectorAll('th');
+const arrayOfEmployees = [];
+const personsAttributes = [];
+
+for (const value of th) {
+  personsAttributes.push(value.textContent);
+}
+
+for (const employeeData of tr) {
+  const person = {};
+
+  for (let i = 0; i < employeeData.children.length; i++) {
+    person[personsAttributes[i]] = employeeData.children[i].textContent;
+  }
+
+  arrayOfEmployees.push(person);
+}
```

Comment:
let's wrap each action in separate function, now it's hard to understand what's going on